workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -127,7 +127,7 @@ async def load_indicator_results(
 
     indicator.result.timestamp_oqt = query_result["timestamp_oqt"]
     indicator.result.timestamp_osm = query_result["timestamp_osm"]
-    indicator.result.label = query_result["result_label"]
+    indicator.result.class_ = query_result["result_class_"]
```

Comment:
If we wouldn't change `class_` (see earlier comment), do you really want to use `class_` in the `query_result` dict?

Reply:
I kept the internal python attribute name `class_` and changed it to `class` for all names outside of the python code (e.g. database or geojson field name)